geniza/pages/models.py

```diff
@@ -16,11 +16,11 @@ class HomePage(Page):
             "h3",
             "bold",
             "italic",
+            "underline",
```

Comment:
Are these two lists of features the same (for home page and content page)? can we define them once and reuse?

Reply:
Yes! Actually will be doing that in work on #581, so I'll go ahead and merge this first.